src/extract.ts

```diff
@@ -17,12 +18,12 @@ interface GitHubUser {
 }
 
 interface Commit {
-    author: GitHubUser;
-    committer: GitHubUser;
+    author?: GitHubUser;
+    committer?: GitHubUser;
     distinct?: unknown; // Unused
     id: string;
-    message: string;
-    timestamp: string;
+    message?: string;
+    timestamp?: string;
```

Comment:
These values are not considered to be optional. Please retrieve them from Git command.

Reply:
Getting committer username might be tough since git only has the email and name. What do you want me to do here?